src/libserver/cfg_utils.c

```diff
@@ -816,7 +816,7 @@ rspamd_config_post_load (struct rspamd_config *cfg,
 	rspamd_regexp_library_init (cfg);
 	rspamd_multipattern_library_init (cfg->hs_cache_dir);
 
-#ifdef WITH_HYPERSCAN
```

Comment:
That will break `cfg->disable_hyperscan` on aarch64.

Reply:
If I read it correctly, it reads as:
If hyperscan not disabled and cpu does not have ssse3 then disable hyperscan

what would be the correct logic for aarch64/neon?  I think all aarch64 chips have neon.